src/patient-registration/section/death-info/death-info-section.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Input } from '../../input/basic-input/input/input.component';
+import { SelectInput } from '../../input/basic-input/select/select-input.component';
+import styles from './../section.css';
+import { FormValues } from '../../patient-registration.component';
+
+interface DeathInfoSectionProps {
+  values: FormValues;
+}
+
+export const DeathInfoSection: React.FC<DeathInfoSectionProps> = ({ values }) => {
+  return (
+    <section className={styles.formSection} aria-label="Death Info Section">
+      <h5 className={`omrs-type-title-5 ${styles.formSectionTitle}`}>Death Info</h5>
+      <section className={styles.fieldGroup}>
+        <Input type="checkbox" label="Is Dead" name="isDead" />
+        {values.isDead && <Input type="date" label="Date of Death" name="deathDate" />}
+        {values.isDead && (
```

Comment:
Instead of having twice the condition you could also use a fragment

```jsx
{value.isDead && (
  <>
   <Input type="date" label="Date of Death" name="deathDate" />
   <SelectInput options={['Unknown cause of death', 'stroke']} label="Cause of Death" name="deathCause" />
  </>
)}
```

Reply:
That is great advice, thanks! Just applied this in a follow-up commit.